src/components/mortgage-investment-compare/input-card.js

```diff
@@ -32,13 +34,18 @@ const InputCard = props => {
     <Card className='p-4'>
       <Form>
         <FormGroup className='mb-3'>
-          <label>How much is your loan?</label>
+          <Tooltip
+            children={<label>How much is your loan?</label>}
+            icon={faQuestionCircle}
+            content='The amount you promise to pay back after making your down payment'
+            id='loanAmt'
+          />
           <InputGroup className='my-2'>
             <InputGroupAddon addonType='prepend'>
               <InputGroupText>$</InputGroupText>
             </InputGroupAddon>
             <ValidatedNumberInput
-              max={500000000}
+              max={5000000000}
```

Comment:
I really wanted to buy the Buckingham Palace. No issues from having an extra 0, our app handles it gracefully